inc/avainsanarivi.inc

```diff
@@ -1074,6 +1075,24 @@ if ($avain_sel["HAE_JA_SELAA"] == "SELECTED" and $_selite) {
   $jatko = 0;
 }
 
+if ($avain_sel["INVLISTA_OSTRY"] == "SELECTED" and $_selite) {
+  $sel = $trow[$i] == "B" ? "selected" : "";
+
+  $ulos = "<td><select name = '{$nimi}'>";
+
+  $ulos .= "<option value = 'A'>";
+  $ulos .= t("Inventointilista -ohjelmassa k�ytet��n monivalintalaatikoita");
```

Comment:
@samiluukkonen @tonyselkamo Näitä voisi ehkä hieman selkiyttää? "monivalintalaatikoita" ja "osasto / try valintaa", mikä on varsinainen ero?